### ISSUE TITLE
No sound MX-23 "Libretto" beta 2 (Chromebook lenovo 11e Glimmer)

### ISSUE THREAD

**Issue by shur601** [WeirdTreeThing/chromebook-linux-audio#10]
In live mode and after installation the sound was not loud but clear, after rebooting  became a bit hoarse. After installation chromebook-linux-audio -  no sound.

**Comment by WeirdTreeThing** [comment]
Attach your dmesg log here.

**Comment by shur601** [comment]
[    0.000000] Linux version 6.1.0-9-amd64 (debian-kernel@lists.debian.org) (gcc-12 (Debian 12.2.0-14) 12.2.0, GNU ld (GNU Binutils for Debian) 2.40) #1 SMP PREEMPT_DYNAMIC Debian 6.1.27-1 (2023-05-08)
[    0.000000] Command line: BOOT_IMAGE=/boot/vmlinuz-6.1.0-9-amd64 root=UUID=ed1e174b-1728-4be8-aef1-94ca2b944ddc ro quiet splash
[    0.000000] x86/fpu: x87 FPU will use FXSAVE
[    0.000000] signal: max sigframe size: 1440
[    0.000000] BIOS-provided physical RAM map:
[    0.000000] BIOS-e820: [mem 0x0000000000000000-0x0000000000000fff] reserved
[    0.000000] BIOS-e820: [mem 0x0000000000001000-0x000000000009ffff] usable
[    0.000000] BIOS-e820: [mem 0x00000000000a0000-0x00000000000fffff] reserved
[    0.000000] BIOS-e820: [mem 0x0000000000100000-0x000000001ec48fff] usable
[    0.000000] BIOS-e820: [mem 0x000000001ec49000-0x000000001ec49fff] ACPI data
[    0.000000] BIOS-e820: [mem 0x000000001ec4a000-0x000000001fc8bfff] usable
[    0.000000] BIOS-e820: [mem 0x000000001fc8c000-0x000000001fc90fff] ACPI data
[    0.000000] BIOS-e820: [mem 0x000000001fc91000-0x000000001fcb6fff] usable
[    0.000000] BIOS-e820: [mem 0x000000001fcb7000-0x000000001fcbafff] reserved
[    0.000000] BIOS-e820: [mem 0x000000001fcbb000-0x000000001feaefff] usable
[    0.000000] BIOS-e820: [mem 0x000000001feaf000-0x000000001ff2efff] type 20
[    0.000000] BIOS-e820: [mem 0x000000001ff2f000-0x000000001fff2fff] reserved
[    0.000000] BIOS-e820: [mem 0x000000001fff3000-0x000000001fff6fff] ACPI NVS
[    0.000000] BIOS-e820: [mem 0x000000001fff7000-0x000000001fffefff] ACPI data
[    0.000000] BIOS-e820: [mem 0x000000001ffff000-0x000000001fffffff] usable
[    0.000000] BIOS-e820: [mem 0x0000000020000000-0x00000000200fffff] reserved
[    0.000000] BIOS-e820: [mem 0x0000000020100000-0x000000007af14fff] usable
[    0.000000] BIOS-e820: [mem 0x000000007af15000-0x000000007fffffff] reserved
[    0.000000] BIOS-e820: [mem 0x00000000ffdc0000-0x00000000ffdfffff] reserved
[    0.000000] BIOS-e820: [mem 0x0000000100000000-0x000000017fffffff] usable
[    0.000000] NX (Execute Disable) protection: active
[    0.000000] efi: EFI v2.70 by EDK II
[    0.000000] efi: SMBIOS=0x1ffd6000 SMBIOS 3.0=0x1ffd4000 ACPI=0x1fffe000 ACPI 2.0=0x1fffe014 MEMATTR=0x1e77d698 
[    0.000000] secureboot: Secure boot disabled
[    0.000000] SMBIOS 3.0.0 present.
[    0.000000] DMI: GOOGLE Glimmer/Glimmer, BIOS MrChromebox-4.19.1 01/29/2023
[    0.000000] tsc: Detected 1833.333 MHz processor
[    0.000026] e820: update [mem 0x00000000-0x00000fff] usable ==> reserved
[    0.000036] e820: remove [mem 0x000a0000-0x000fffff] usable
[    0.000055] last_pfn = 0x180000 max_arch_pfn = 0x400000000
[    0.000117] x86/PAT: Configuration [0-7]: WB  WC  UC- UC  WB  WP  UC- WT  
[    0.000364] last_pfn = 0x7af15 max_arch_pfn = 0x400000000
[    0.010259] RAMDISK: [mem 0x354f5000-0x36a71fff]
[    0.010273] ACPI: Early table checksum verification disabled
[    0.010280] ACPI: RSDP 0x000000001FFFE014 000024 (v02 COREv4)
[    0.010291] ACPI: XSDT 0x000000001FFFD0E8 00005C (v01 COREv4 COREBOOT 00000000      01000013)
[    0.010304] ACPI: FACP 0x000000001FFFC000 000114 (v06 COREv4 COREBOOT 00000000 CORE 20220331)
[    0.010318] ACPI: DSDT 0x000000001FFF8000 003FC9 (v02 COREv4 COREBOOT 20110725 INTL 20220331)
[    0.010329] ACPI: FACS 0x000000007AF2D240 000040
[    0.010337] ACPI: SSDT 0x000000001FC8F000 0016A6 (v02 COREv4 COREBOOT 0000002A CORE 20220331)
[    0.010347] ACPI: MCFG 0x000000001FFF7000 00003C (v01 COREv4 COREBOOT 00000000 CORE 20220331)
[    0.010357] ACPI: TCPA 0x000000001FC8E000 000032 (v02 COREv4 COREBOOT 00000000 CORE 20220331)
[    0.010367] ACPI: APIC 0x000000001FC8D000 000072 (v03 COREv4 COREBOOT 00000000 CORE 20220331)
[    0.010377] ACPI: HPET 0x000000001FC8C000 000038 (v01 COREv4 COREBOOT 00000000 CORE 20220331)
[    0.010386] ACPI: BGRT 0x000000001EC49000 000038 (v01 INTEL  EDK2     00000002      01000013)
[    0.010395] ACPI: Reserving FACP table memory at [mem 0x1fffc000-0x1fffc113]
[    0.010399] ACPI: Reserving DSDT table memory at [mem 0x1fff8000-0x1fffbfc8]
[    0.010403] ACPI: Reserving FACS table memory at [mem 0x7af2d240-0x7af2d27f]
[    0.010406] ACPI: Reserving SSDT table memory at [mem 0x1fc8f000-0x1fc906a5]
[    0.010409] ACPI: Reserving MCFG table memory at [mem 0x1fff7000-0x1fff703b]
[    0.010413] ACPI: Reserving TCPA table memory at [mem 0x1fc8e000-0x1fc8e031]
[    0.010416] ACPI: Reserving APIC table memory at [mem 0x1fc8d000-0x1fc8d071]
[    0.010419] ACPI: Reserving HPET table memory at [mem 0x1fc8c000-0x1fc8c037]
[    0.010423] ACPI: Reserving BGRT table memory at [mem 0x1ec49000-0x1ec49037]
[    0.010523] No NUMA configuration found
[    0.010527] Faking a node at [mem 0x0000000000000000-0x000000017fffffff]
[    0.010551] NODE_DATA(0) allocated [mem 0x17ffd2000-0x17fffcfff]
[    0.011338] Zone ranges:
[    0.011341]   DMA      [mem 0x0000000000001000-0x0000000000ffffff]
[    0.011346]   DMA32    [mem 0x0000000001000000-0x00000000ffffffff]
[    0.011351]   Normal   [mem 0x0000000100000000-0x000000017fffffff]
[    0.011356]   Device   empty
[    0.011359] Movable zone start for each node
[    0.011365] Early memory node ranges
[    0.011367]   node   0: [mem 0x0000000000001000-0x000000000009ffff]
[    0.011371]   node   0: [mem 0x0000000000100000-0x000000001ec48fff]
[    0.011375]   node   0: [mem 0x000000001ec4a000-0x000000001fc8bfff]
[    0.011378]   node   0: [mem 0x000000001fc91000-0x000000001fcb6fff]
[    0.011381]   node   0: [mem 0x000000001fcbb000-0x000000001feaefff]
[    0.011384]   node   0: [mem 0x000000001ffff000-0x000000001fffffff]
[    0.011388]   node   0: [mem 0x0000000020100000-0x000000007af14fff]
[    0.011391]   node   0: [mem 0x0000000100000000-0x000000017fffffff]
[    0.011396] Initmem setup node 0 [mem 0x0000000000001000-0x000000017fffffff]
[    0.011409] On node 0, zone DMA: 1 pages in unavailable ranges
[    0.011492] On node 0, zone DMA: 96 pages in unavailable ranges
[    0.014144] On node 0, zone DMA32: 1 pages in unavailable ranges
[    0.014150] On node 0, zone DMA32: 5 pages in unavailable ranges
[    0.014164] On node 0, zone DMA32: 4 pages in unavailable ranges
[    0.014177] On node 0, zone DMA32: 336 pages in unavailable ranges
[    0.022031] On node 0, zone DMA32: 256 pages in unavailable ranges
[    0.023419] On node 0, zone Normal: 20715 pages in unavailable ranges
[    0.023428] x86/hpet: Will disable the HPET for this platform because it's not reliable
[    0.023436] Reserving Intel graphics memory at [mem 0x7c000000-0x7fffffff]
[    0.023535] ACPI: PM-Timer IO Port: 0x408
[    0.023552] ACPI: LAPIC_NMI (acpi_id[0xff] high edge lint[0x1])
[    0.023571] IOAPIC[0]: apic_id 0, version 32, address 0xfec00000, GSI 0-86
[    0.023579] ACPI: INT_SRC_OVR (bus 0 bus_irq 0 global_irq 2 dfl dfl)
[    0.023584] ACPI: INT_SRC_OVR (bus 0 bus_irq 9 global_irq 9 high level)
[    0.023594] ACPI: Using ACPI (MADT) for SMP configuration information
[    0.023598] ACPI: HPET id: 0x8086a201 base: 0xfed00000
[    0.023621] e820: update [mem 0x1e737000-0x1e75bfff] usable ==> reserved
[    0.023648] TSC deadline timer available
[    0.023651] smpboot: Allowing 4 CPUs, 0 hotplug CPUs
[    0.023704] PM: hibernation: Registered nosave memory: [mem 0x00000000-0x00000fff]
[    0.023711] PM: hibernation: Registered nosave memory: [mem 0x000a0000-0x000fffff]
[    0.023717] PM: hibernation: Registered nosave memory: [mem 0x1e737000-0x1e75bfff]
[    0.023723] PM: hibernation: Registered nosave memory: [mem 0x1ec49000-0x1ec49fff]
[    0.023728] PM: hibernation: Registered nosave memory: [mem 0x1fc8c000-0x1fc90fff]
[    0.023734] PM: hibernation: Registered nosave memory: [mem 0x1fcb7000-0x1fcbafff]
[    0.023739] PM: hibernation: Registered nosave memory: [mem 0x1feaf000-0x1ff2efff]
[    0.023742] PM: hibernation: Registered nosave memory: [mem 0x1ff2f000-0x1fff2fff]
[    0.023745] PM: hibernation: Registered nosave memory: [mem 0x1fff3000-0x1fff6fff]
[    0.023748] PM: hibernation: Registered nosave memory: [mem 0x1fff7000-0x1fffefff]
[    0.023753] PM: hibernation: Registered nosave memory: [mem 0x20000000-0x200fffff]
[    0.023759] PM: hibernation: Registered nosave memory: [mem 0x7af15000-0x7fffffff]
[    0.023762] PM: hibernation: Registered nosave memory: [mem 0x80000000-0xffdbffff]
[    0.023764] PM: hibernation: Registered nosave memory: [mem 0xffdc0000-0xffdfffff]
[    0.023767] PM: hibernation: Registered nosave memory: [mem 0xffe00000-0xffffffff]
[    0.023772] [mem 0x80000000-0xffdbffff] available for PCI devices
[    0.023776] Booting paravirtualized kernel on bare hardware
[    0.023781] clocksource: refined-jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 7645519600211568 ns
[    0.033226] setup_percpu: NR_CPUS:8192 nr_cpumask_bits:4 nr_cpu_ids:4 nr_node_ids:1
[    0.034059] percpu: Embedded 61 pages/cpu s212992 r8192 d28672 u524288
[    0.034082] pcpu-alloc: s212992 r8192 d28672 u524288 alloc=1*2097152
[    0.034090] pcpu-alloc: [0] 0 1 2 3 
[    0.034178] Fallback order for Node 0: 0 
[    0.034187] Built 1 zonelists, mobility grouping on.  Total pages: 1009528
[    0.034192] Policy zone: Normal
[    0.034195] Kernel command line: BOOT_IMAGE=/boot/vmlinuz-6.1.0-9-amd64 root=UUID=ed1e174b-1728-4be8-aef1-94ca2b944ddc ro quiet splash
[    0.034317] Unknown kernel command line parameters "splash BOOT_IMAGE=/boot/vmlinuz-6.1.0-9-amd64", will be passed to user space.
[    0.034372] random: crng init done
[    0.035540] Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes, linear)
[    0.036152] Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes, linear)
[    0.036238] mem auto-init: stack:all(zero), heap alloc:on, heap free:off
[    0.036253] software IO TLB: area num 4.
[    0.096888] Memory: 1971684K/4108648K available (14342K kernel code, 2324K rwdata, 8728K rodata, 2772K init, 17436K bss, 244308K reserved, 0K cma-reserved)
[    0.097366] SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
[    0.097407] Kernel/User page tables isolation: enabled
[    0.097455] ftrace: allocating 39983 entries in 157 pages
[    0.108168] ftrace: allocated 157 pages with 5 groups
[    0.109348] Dynamic Preempt: voluntary
[    0.109434] rcu: Preemptible hierarchical RCU implementation.
[    0.109437] rcu: 	RCU restricting CPUs from NR_CPUS=8192 to nr_cpu_ids=4.
[    0.109442] 	Trampoline variant of Tasks RCU enabled.
[    0.109444] 	Rude variant of Tasks RCU enabled.
[    0.109445] 	Tracing variant of Tasks RCU enabled.
[    0.109447] rcu: RCU calculated value of scheduler-enlistment delay is 25 jiffies.
[    0.109450] rcu: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=4
[    0.122320] NR_IRQS: 524544, nr_irqs: 1024, preallocated irqs: 16
[    0.122594] rcu: srcu_init: Setting srcu_struct sizes based on contention.
[    0.122713] Console: colour dummy device 80x25
[    0.122760] printk: console [tty0] enabled
[    0.122812] ACPI: Core revision 20220331
[    0.122955] APIC: Switch to symmetric I/O mode setup
[    0.123799] clocksource: tsc-early: mask: 0xffffffffffffffff max_cycles: 0x34da54753aa, max_idle_ns: 881590618787 ns
[    0.123819] Calibrating delay loop (skipped), value calculated using timer frequency.. 3666.66 BogoMIPS (lpj=7333332)
[    0.123828] pid_max: default: 32768 minimum: 301
[    0.127809] LSM: Security Framework initializing
[    0.127809] landlock: Up and running.
[    0.127809] Yama: disabled by default; enable with sysctl kernel.yama.*
[    0.127809] AppArmor: AppArmor initialized
[    0.127809] TOMOYO Linux initialized
[    0.127809] LSM support for eBPF active
[    0.127809] Mount-cache hash table entries: 8192 (order: 4, 65536 bytes, linear)
[    0.127809] Mountpoint-cache hash table entries: 8192 (order: 4, 65536 bytes, linear)
[    0.127809] CPU0: Thermal monitoring enabled (TM1)
[    0.127809] process: using mwait in idle threads
[    0.127809] Last level iTLB entries: 4KB 48, 2MB 0, 4MB 0
[    0.127809] Last level dTLB entries: 4KB 128, 2MB 16, 4MB 16, 1GB 0
[    0.127809] Spectre V1 : Mitigation: usercopy/swapgs barriers and __user pointer sanitization
[    0.127809] Spectre V2 : Mitigation: Retpolines
[    0.127809] Spectre V2 : Spectre v2 / SpectreRSB mitigation: Filling RSB on context switch
[    0.127809] Spectre V2 : Spectre v2 / SpectreRSB : Filling RSB on VMEXIT
[    0.127809] Spectre V2 : Enabling Restricted Speculation for firmware calls
[    0.127809] Spectre V2 : mitigation: Enabling conditional Indirect Branch Prediction Barrier
[    0.127809] MDS: Mitigation: Clear CPU buffers
[    0.127809] MMIO Stale Data: Unknown: No mitigations
[    0.127809] Freeing SMP alternatives memory: 36K
[    0.127809] smpboot: CPU0: Intel(R) Celeron(R) CPU  N2940  @ 1.83GHz (family: 0x6, model: 0x37, stepping: 0x8)
[    0.127809] cblist_init_generic: Setting adjustable number of callback queues.
[    0.127809] cblist_init_generic: Setting shift to 2 and lim to 1.
[    0.127809] cblist_init_generic: Setting shift to 2 and lim to 1.
[    0.127809] cblist_init_generic: Setting shift to 2 and lim to 1.
[    0.127809] Performance Events: PEBS fmt2+, 8-deep LBR, Silvermont events, 8-deep LBR, full-width counters, Intel PMU driver.
[    0.127809] ... version:                3
[    0.127809] ... bit width:              40
[    0.127809] ... generic registers:      2
[    0.127809] ... value mask:             000000ffffffffff
[    0.127809] ... max period:             0000007fffffffff
[    0.127809] ... fixed-purpose events:   3
[    0.127809] ... event mask:             0000000700000003
[    0.127809] Estimated ratio of average max frequency by base frequency (times 1024): 1256
[    0.127809] rcu: Hierarchical SRCU implementation.
[    0.127809] rcu: 	Max phase no-delay instances is 1000.
[    0.127809] NMI watchdog: Enabled. Permanently consumes one hw-PMU counter.
[    0.127809] smp: Bringing up secondary CPUs ...
[    0.127809] x86: Booting SMP configuration:
[    0.127809] .... node  #0, CPUs:      #1 #2 #3
[    0.128392] smp: Brought up 1 node, 4 CPUs
[    0.128392] smpboot: Max logical packages: 1
[    0.128392] smpboot: Total of 4 processors activated (14666.66 BogoMIPS)
[    0.136829] node 0 deferred pages initialised in 8ms
[    0.136853] devtmpfs: initialized
[    0.136853] x86/mm: Memory block size: 128MB
[    0.137507] ACPI: PM: Registering ACPI NVS region [mem 0x1fff3000-0x1fff6fff] (16384 bytes)
[    0.137507] clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 7645041785100000 ns
[    0.139843] futex hash table entries: 1024 (order: 4, 65536 bytes, linear)
[    0.139968] pinctrl core: initialized pinctrl subsystem
[    0.142130] NET: Registered PF_NETLINK/PF_ROUTE protocol family
[    0.142564] DMA: preallocated 512 KiB GFP_KERNEL pool for atomic allocations
[    0.142729] DMA: preallocated 512 KiB GFP_KERNEL|GFP_DMA pool for atomic allocations
[    0.142897] DMA: preallocated 512 KiB GFP_KERNEL|GFP_DMA32 pool for atomic allocations
[    0.142945] audit: initializing netlink subsys (disabled)
[    0.142995] audit: type=2000 audit(1687978321.016:1): state=initialized audit_enabled=0 res=1
[    0.142995] thermal_sys: Registered thermal governor 'fair_share'
[    0.142995] thermal_sys: Registered thermal governor 'bang_bang'
[    0.142995] thermal_sys: Registered thermal governor 'step_wise'
[    0.142995] thermal_sys: Registered thermal governor 'user_space'
[    0.142995] thermal_sys: Registered thermal governor 'power_allocator'
[    0.142995] cpuidle: using governor ladder
[    0.142995] cpuidle: using governor menu
[    0.142995] acpiphp: ACPI Hot Plug PCI Controller Driver version: 0.5
[    0.143888] PCI: MMCONFIG for domain 0000 [bus 00-ff] at [mem 0xe0000000-0xefffffff] (base 0xe0000000)
[    0.143902] PCI: not using MMCONFIG
[    0.143912] PCI: Using configuration type 1 for base access
[    0.144295] ENERGY_PERF_BIAS: Set to 'normal', was 'performance'
[    0.146969] kprobes: kprobe jump-optimization is enabled. All kprobes are optimized if possible.
[    0.148212] HugeTLB: registered 2.00 MiB page size, pre-allocated 0 pages
[    0.148219] HugeTLB: 28 KiB vmemmap can be freed for a 2.00 MiB page
[    0.149346] ACPI: Added _OSI(Module Device)
[    0.149353] ACPI: Added _OSI(Processor Device)
[    0.149357] ACPI: Added _OSI(3.0 _SCP Extensions)
[    0.149360] ACPI: Added _OSI(Processor Aggregator Device)
[    0.152315] ACPI: 2 ACPI AML tables successfully acquired and loaded
[    0.187122] ACPI: EC: EC started
[    0.187122] ACPI: EC: interrupt blocked
[    0.187122] ACPI: EC: EC_CMD/EC_SC=0x66, EC_DATA=0x62
[    0.187122] ACPI: \_SB_.PCI0.LPCB.EC0_: Boot DSDT EC used to handle transactions
[    0.187122] ACPI: Interpreter enabled
[    0.187122] ACPI: PM: (supports S0 S3 S4 S5)
[    0.187122] ACPI: Using IOAPIC for interrupt routing
[    0.187122] PCI: MMCONFIG for domain 0000 [bus 00-ff] at [mem 0xe0000000-0xefffffff] (base 0xe0000000)
[    0.187122] PCI: MMCONFIG at [mem 0xe0000000-0xefffffff] reserved in ACPI motherboard resources
[    0.187122] PCI: Using host bridge windows from ACPI; if necessary, use "pci=nocrs" and report a bug
[    0.187122] PCI: Ignoring E820 reservations for host bridge windows
[    0.195646] ACPI: PM: Power Resource [PLPE]
[    0.198287] ACPI: PCI Root Bridge [PCI0] (domain 0000 [bus 00-ff])
[    0.198304] acpi PNP0A08:00: _OSC: OS supports [ExtendedConfig ASPM ClockPM Segments MSI HPX-Type3]
[    0.198387] acpi PNP0A08:00: _OSC: OS now controls [PCIeHotplug SHPCHotplug PME AER PCIeCapability LTR]
[    0.199164] PCI host bridge to bus 0000:00
[    0.199171] pci_bus 0000:00: root bus resource [io  0x0000-0x0cf7 window]
[    0.199178] pci_bus 0000:00: root bus resource [io  0x0d00-0xffff window]
[    0.199184] pci_bus 0000:00: root bus resource [mem 0x000a0000-0x000fffff window]
[    0.199189] pci_bus 0000:00: root bus resource [mem 0x20000000-0x200fffff window]
[    0.199194] pci_bus 0000:00: root bus resource [mem 0x80000000-0xdfffffff window]
[    0.199200] pci_bus 0000:00: root bus resource [mem 0xfed40000-0xfed44fff window]
[    0.199206] pci_bus 0000:00: root bus resource [bus 00-ff]
[    0.199243] pci 0000:00:00.0: [8086:0f00] type 00 class 0x060000
[    0.199400] pci 0000:00:02.0: [8086:0f31] type 00 class 0x030000
[    0.199417] pci 0000:00:02.0: reg 0x10: [mem 0x90000000-0x903fffff]
[    0.199431] pci 0000:00:02.0: reg 0x18: [mem 0x80000000-0x8fffffff pref]
[    0.199445] pci 0000:00:02.0: reg 0x20: [io  0x1000-0x1007]
[    0.199470] pci 0000:00:02.0: BAR 2: assigned to efifb
[    0.199479] pci 0000:00:02.0: Video device with shadowed ROM at [mem 0x000c0000-0x000dffff]
[    0.199705] pci 0000:00:14.0: [8086:0f35] type 00 class 0x0c0330
[    0.199727] pci 0000:00:14.0: reg 0x10: [mem 0x90700000-0x9070ffff 64bit]
[    0.199797] pci 0000:00:14.0: PME# supported from D3hot D3cold
[    0.200002] pci 0000:00:1b.0: [8086:0f04] type 00 class 0x040300
[    0.200028] pci 0000:00:1b.0: reg 0x10: [mem 0x90714000-0x90717fff 64bit]
[    0.200110] pci 0000:00:1b.0: PME# supported from D0 D3hot D3cold
[    0.200230] pci 0000:00:1c.0: [8086:0f48] type 01 class 0x060400
[    0.200315] pci 0000:00:1c.0: PME# supported from D0 D3hot D3cold
[    0.200542] pci 0000:00:1f.0: [8086:0f1c] type 00 class 0x060100
[    0.200962] pci 0000:01:00.0: [8086:08b2] type 00 class 0x028000
[    0.201018] pci 0000:01:00.0: reg 0x10: [mem 0x90600000-0x90601fff 64bit]
[    0.201264] pci 0000:01:00.0: PME# supported from D0 D3hot D3cold
[    0.201624] pci 0000:00:1c.0: PCI bridge to [bus 01]
[    0.201634] pci 0000:00:1c.0:   bridge window [mem 0x90600000-0x906fffff]
[    0.201864] ACPI: PCI: Interrupt link LNKA configured for IRQ 0
[    0.202002] ACPI: PCI: Interrupt link LNKB configured for IRQ 0
[    0.202137] ACPI: PCI: Interrupt link LNKC configured for IRQ 0
[    0.202273] ACPI: PCI: Interrupt link LNKD configured for IRQ 0
[    0.202414] ACPI: PCI: Interrupt link LNKE configured for IRQ 0
[    0.202558] ACPI: PCI: Interrupt link LNKF configured for IRQ 0
[    0.202694] ACPI: PCI: Interrupt link LNKG configured for IRQ 0
[    0.202836] ACPI: PCI: Interrupt link LNKH configured for IRQ 0
[    0.212532] ACPI: EC: interrupt unblocked
[    0.212540] ACPI: EC: event unblocked
[    0.212546] ACPI: EC: EC_CMD/EC_SC=0x66, EC_DATA=0x62
[    0.212550] ACPI: EC: GPE=0x18
[    0.212555] ACPI: \_SB_.PCI0.LPCB.EC0_: Boot DSDT EC initialization complete
[    0.212561] ACPI: \_SB_.PCI0.LPCB.EC0_: EC: Used to handle transactions and events
[    0.212847] iommu: Default domain type: Translated 
[    0.212847] iommu: DMA domain TLB invalidation policy: lazy mode 
[    0.283380] EDAC MC: Ver: 3.0.0
[    0.283915] Registered efivars operations
[    0.284320] NetLabel: Initializing
[    0.284325] NetLabel:  domain hash size = 128
[    0.284328] NetLabel:  protocols = UNLABELED CIPSOv4 CALIPSO
[    0.284370] NetLabel:  unlabeled traffic allowed by default
[    0.284375] PCI: Using ACPI for IRQ routing
[    0.290658] PCI: pci_cache_line_size set to 64 bytes
[    0.290709] e820: reserve RAM buffer [mem 0x1e737000-0x1fffffff]
[    0.290716] e820: reserve RAM buffer [mem 0x1ec49000-0x1fffffff]
[    0.290720] e820: reserve RAM buffer [mem 0x1fc8c000-0x1fffffff]
[    0.290725] e820: reserve RAM buffer [mem 0x1fcb7000-0x1fffffff]
[    0.290728] e820: reserve RAM buffer [mem 0x1feaf000-0x1fffffff]
[    0.290732] e820: reserve RAM buffer [mem 0x7af15000-0x7bffffff]
[    0.291832] pci 0000:00:02.0: vgaarb: setting as boot VGA device
[    0.291839] pci 0000:00:02.0: vgaarb: bridge control possible
[    0.291843] pci 0000:00:02.0: vgaarb: VGA device added: decodes=io+mem,owns=io+mem,locks=none
[    0.291849] vgaarb: loaded
[    0.292091] clocksource: Switched to clocksource tsc-early
[    0.292467] VFS: Disk quotas dquot_6.6.0
[    0.292503] VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
[    0.292840] AppArmor: AppArmor Filesystem Enabled
[    0.292840] pnp: PnP ACPI init
[    0.293021] system 00:00: [mem 0xfeb00000-0xfebfffff] has been reserved
[    0.293033] system 00:00: [mem 0xe0000000-0xefffffff] has been reserved
[    0.293040] system 00:00: [mem 0xfed03000-0xfed033ff] has been reserved
[    0.293046] system 00:00: [mem 0xfed08000-0xfed083ff] has been reserved
[    0.293053] system 00:00: [mem 0xfed01000-0xfed013ff] has been reserved
[    0.293059] system 00:00: [mem 0xfef00000-0xfeffffff] has been reserved
[    0.293066] system 00:00: [mem 0xfed05000-0xfed057ff] has been reserved
[    0.293072] system 00:00: [mem 0xfed1c000-0xfed1c3ff] has been reserved
[    0.293632] system 00:01: [mem 0xfed00000-0xfed003ff] has been reserved
[    0.293919] system 00:04: [io  0x0900-0x09fe] has been reserved
[    0.294035] system 00:05: [io  0x0200] has been reserved
[    0.294043] system 00:05: [io  0x0204] has been reserved
[    0.294048] system 00:05: [io  0x0800-0x087f] has been reserved
[    0.294054] system 00:05: [io  0x0880-0x08ff] has been reserved
[    0.294991] pnp: PnP ACPI: found 8 devices
[    0.305149] clocksource: acpi_pm: mask: 0xffffff max_cycles: 0xffffff, max_idle_ns: 2085701024 ns
[    0.305338] NET: Registered PF_INET protocol family
[    0.305555] IP idents hash table entries: 65536 (order: 7, 524288 bytes, linear)
[    0.308654] tcp_listen_portaddr_hash hash table entries: 2048 (order: 3, 32768 bytes, linear)
[    0.308774] Table-perturb hash table entries: 65536 (order: 6, 262144 bytes, linear)
[    0.308893] TCP established hash table entries: 32768 (order: 6, 262144 bytes, linear)
[    0.309290] TCP bind hash table entries: 32768 (order: 8, 1048576 bytes, linear)
[    0.309635] TCP: Hash tables configured (established 32768 bind 32768)
[    0.309872] MPTCP token hash table entries: 4096 (order: 4, 98304 bytes, linear)
[    0.309953] UDP hash table entries: 2048 (order: 4, 65536 bytes, linear)
[    0.310013] UDP-Lite hash table entries: 2048 (order: 4, 65536 bytes, linear)
[    0.310152] NET: Registered PF_UNIX/PF_LOCAL protocol family
[    0.310176] NET: Registered PF_XDP protocol family
[    0.310201] pci 0000:00:1c.0: PCI bridge to [bus 01]
[    0.310213] pci 0000:00:1c.0:   bridge window [mem 0x90600000-0x906fffff]
[    0.310225] pci_bus 0000:00: resource 4 [io  0x0000-0x0cf7 window]
[    0.310231] pci_bus 0000:00: resource 5 [io  0x0d00-0xffff window]
[    0.310236] pci_bus 0000:00: resource 6 [mem 0x000a0000-0x000fffff window]
[    0.310241] pci_bus 0000:00: resource 7 [mem 0x20000000-0x200fffff window]
[    0.310246] pci_bus 0000:00: resource 8 [mem 0x80000000-0xdfffffff window]
[    0.310251] pci_bus 0000:00: resource 9 [mem 0xfed40000-0xfed44fff window]
[    0.310256] pci_bus 0000:01: resource 1 [mem 0x90600000-0x906fffff]
[    0.311694] PCI: CLS 64 bytes, default 64
[    0.311719] PCI-DMA: Using software bounce buffering for IO (SWIOTLB)
[    0.311722] software IO TLB: mapped [mem 0x0000000076f15000-0x000000007af15000] (64MB)
[    0.311802] clocksource: tsc: mask: 0xffffffffffffffff max_cycles: 0x34da54753aa, max_idle_ns: 881590618787 ns
[    0.311909] clocksource: Switched to clocksource tsc
[    0.311977] Trying to unpack rootfs image as initramfs...
[    0.323618] Initialise system trusted keyrings
[    0.323645] Key type blacklist registered
[    0.323757] workingset: timestamp_bits=36 max_order=20 bucket_order=0
[    0.327758] zbud: loaded
[    0.328456] integrity: Platform Keyring initialized
[    0.328474] integrity: Machine keyring initialized
[    0.328479] Key type asymmetric registered
[    0.328483] Asymmetric key parser 'x509' registered
[    0.827498] Freeing initrd memory: 22004K
[    0.844680] alg: self-tests for CTR-KDF (hmac(sha256)) passed
[    0.844747] Block layer SCSI generic (bsg) driver version 0.4 loaded (major 250)
[    0.844862] io scheduler mq-deadline registered
[    0.848397] pcieport 0000:00:1c.0: PME: Signaling with IRQ 87
[    0.848582] pcieport 0000:00:1c.0: AER: enabled with IRQ 87
[    0.848757] shpchp: Standard Hot Plug PCI Controller Driver version: 0.4
[    0.852359] Serial: 8250/16550 driver, 4 ports, IRQ sharing enabled
[    0.853945] hpet_acpi_add: no address or irqs in _CRS
[    0.853990] Linux agpgart interface v0.103
[    0.858090] tpm_tis 00:07: 1.2 TPM (device-id 0xB, rev-id 16)
[    0.882489] AMD-Vi: AMD IOMMUv2 functionality not available on this system - This is not a bug.
[    0.884784] i8042: PNP: PS/2 Controller [PNP0303:PS2K] at 0x60,0x64 irq 53
[    0.884794] i8042: PNP: PS/2 appears to have AUX port disabled, if this is incorrect please boot with i8042.nopnp
[    0.885894] i8042: Warning: Keylock active
[    0.886053] serio: i8042 KBD port at 0x60,0x64 irq 53
[    0.886536] mousedev: PS/2 mouse device common for all mice
[    0.886639] rtc_cmos 00:03: RTC can wake from S4
[    0.887188] rtc_cmos 00:03: registered as rtc0
[    0.887225] rtc_cmos 00:03: setting system clock to 2023-06-28T18:52:02 UTC (1687978322)
[    0.887312] rtc_cmos 00:03: alarms up to one month, y3k, 242 bytes nvram
[    0.888506] intel_pstate: Intel P-state driver initializing
[    0.890422] input: AT Translated Set 2 keyboard as /devices/platform/i8042/serio0/input/input0
[    0.892993] ledtrig-cpu: registered to indicate activity on CPUs
[    0.893575] efifb: probing for efifb
[    0.893607] efifb: framebuffer at 0x80000000, using 4160k, total 4160k
[    0.893616] efifb: mode is 1366x768x32, linelength=5504, pages=1
[    0.893622] efifb: scrolling: redraw
[    0.893625] efifb: Truecolor: size=8:8:8:8, shift=24:16:8:0
[    0.894015] Console: switching to colour frame buffer device 170x48
[    0.898095] fb0: EFI VGA frame buffer device
[    0.930192] NET: Registered PF_INET6 protocol family
[    0.953232] Segment Routing with IPv6
[    0.953270] In-situ OAM (IOAM) with IPv6
[    0.953324] mip6: Mobile IPv6
[    0.953336] NET: Registered PF_PACKET protocol family
[    0.954198] mpls_gso: MPLS GSO support
[    0.955698] microcode: sig=0x30678, pf=0x8, revision=0x838
[    0.956110] microcode: Microcode Update Driver: v2.2.
[    0.956124] IPI shorthand broadcast: enabled
[    0.956188] sched_clock: Marking stable (954633822, 1185684)->(992699593, -36880087)
[    0.957815] registered taskstats version 1
[    0.957847] Loading compiled-in X.509 certificates
[    1.021324] Loaded X.509 cert 'Debian Secure Boot CA: 6ccece7e4c6c0d1f6149f3dd27dfcc5cbb419ea1'
[    1.021445] Loaded X.509 cert 'Debian Secure Boot Signer 2022 - linux: 14011249c2675ea8e5148542202005810584b25f'
[    1.024355] zswap: loaded using pool lzo/zbud
[    1.027582] Key type .fscrypt registered
[    1.027596] Key type fscrypt-provisioning registered
[    1.051336] Key type encrypted registered
[    1.051355] AppArmor: AppArmor sha1 policy hashing enabled
[    1.051581] ima: Allocated hash algorithm: sha256
[    1.097011] ima: No architecture policies found
[    1.097065] evm: Initialising EVM extended attributes:
[    1.097069] evm: security.selinux
[    1.097073] evm: security.SMACK64 (disabled)
[    1.097076] evm: security.SMACK64EXEC (disabled)
[    1.097078] evm: security.SMACK64TRANSMUTE (disabled)
[    1.097081] evm: security.SMACK64MMAP (disabled)
[    1.097083] evm: security.apparmor
[    1.097086] evm: security.ima
[    1.097088] evm: security.capability
[    1.097091] evm: HMAC attrs: 0x1
[    1.458965] Freeing unused decrypted memory: 2036K
[    1.462792] Freeing unused kernel image (initmem) memory: 2772K
[    1.463635] Write protecting the kernel read-only data: 26624k
[    1.468220] Freeing unused kernel image (text/rodata gap) memory: 2040K
[    1.470959] Freeing unused kernel image (rodata/data gap) memory: 1512K
[    1.561512] x86/mm: Checked W+X mappings: passed, no W+X pages found.
[    1.562045] x86/mm: Checking user space page tables
[    1.625112] x86/mm: Checked W+X mappings: passed, no W+X pages found.
[    1.625125] Run /init as init process
[    1.625128]   with arguments:
[    1.625131]     /init
[    1.625133]     splash
[    1.625135]   with environment:
[    1.625137]     HOME=/
[    1.625139]     TERM=linux
[    1.625141]     BOOT_IMAGE=/boot/vmlinuz-6.1.0-9-amd64
[    2.255134] sdhci: Secure Digital Host Controller Interface driver
[    2.255141] sdhci: Copyright(c) Pierre Ossman
[    2.258577] sdhci-acpi 80860F16:00: failed to setup card detect gpio
[    2.259150] mmc0: SDHCI controller on ACPI [80860F16:00] using ADMA
[    2.276654] mmc1: SDHCI controller on ACPI [80860F14:00] using ADMA
[    2.340853] ACPI: bus type drm_connector registered
[    2.421668] mmc1: new HS200 MMC card at address 0001
[    2.429460] mmcblk1: mmc1:0001 SEM16G 14.7 GiB 
[    2.433091]  mmcblk1: p1 p2 p3 p4
[    2.434010] mmcblk1boot0: mmc1:0001 SEM16G 4.00 MiB 
[    2.435064] mmcblk1boot1: mmc1:0001 SEM16G 4.00 MiB 
[    2.436147] mmcblk1rpmb: mmc1:0001 SEM16G 4.00 MiB, chardev (245:0)
[    3.268606] Console: switching to colour dummy device 80x25
[    3.268944] i915 0000:00:02.0: vgaarb: deactivate vga console
[    3.332720] [drm] Initialized i915 1.6.0 20201103 for 0000:00:02.0 on minor 0
[    3.333485] ACPI: video: Video Device [GFX0] (multi-head: yes  rom: no  post: no)
[    3.333691] input: Video Bus as /devices/LNXSYSTM:00/LNXSYBUS:00/PNP0A08:00/LNXVIDEO:00/input/input1
[    3.338278] fbcon: i915drmfb (fb0) is primary device
[    3.346318] Console: switching to colour frame buffer device 170x48
[    3.373253] i915 0000:00:02.0: [drm] fb0: i915drmfb frame buffer device
[    3.620118] raid6: sse2x4   gen()  5406 MB/s
[    3.688159] raid6: sse2x2   gen()  5251 MB/s
[    3.756156] raid6: sse2x1   gen()  3994 MB/s
[    3.756161] raid6: using algorithm sse2x4 gen() 5406 MB/s
[    3.824155] raid6: .... xor() 1491 MB/s, rmw enabled
[    3.824159] raid6: using ssse3x2 recovery algorithm
[    3.827028] async_tx: api initialized (async)
[    3.830601] xor: measuring software checksum speed
[    3.831758]    prefetch64-sse  :  8555 MB/sec
[    3.833075]    generic_sse     :  7584 MB/sec
[    3.833079] xor: using function: prefetch64-sse (8555 MB/sec)
[   11.003907] EXT4-fs (mmcblk1p4): mounted filesystem with ordered data mode. Quota mode: none.
[   11.230092] Not activating Mandatory Access Control as /sbin/tomoyo-init does not exist.
[   12.837542] pstore: Using crash dump compression: deflate
[   12.838596] input: Lid Switch as /devices/LNXSYSTM:00/LNXSYBUS:00/PNP0A08:00/device:00/PNP0C09:00/PNP0C0D:00/input/input2
[   12.840189] pstore: Registered efi as persistent store backend
[   12.857206] ACPI: button: Lid Switch [LID0]
[   12.875926] pwm-lpss 80860F09:00: can't request region for resource [mem 0x00000000-0x00000fff]
[   12.875942] pwm-lpss: probe of 80860F09:00 failed with error -16
[   12.876438] input: Sleep Button as /devices/LNXSYSTM:00/LNXSYBUS:00/PNP0C0E:00/input/input3
[   12.879794] ACPI: button: Sleep Button [TPAD]
[   12.879986] input: Sleep Button as /devices/LNXSYSTM:00/LNXSYBUS:00/PNP0C0E:01/input/input4
[   12.896124] ACPI: button: Sleep Button [TSCR]
[   12.898967] input: Power Button as /devices/LNXSYSTM:00/LNXPWRBN:00/input/input5
[   12.900618] dw_dmac INTL9C60:00: DesignWare DMA Controller, 8 channels
[   12.915189] ACPI: button: Power Button [PWRF]
[   12.940520] dw_dmac INTL9C60:01: DesignWare DMA Controller, 8 channels
[   12.978999] ACPI: bus type USB registered
[   12.979071] usbcore: registered new interface driver usbfs
[   12.979096] usbcore: registered new interface driver hub
[   12.979132] usbcore: registered new device driver usb
[   13.030447] ACPI: AC: AC Adapter [AC] (off-line)
[   13.031674] ACPI: battery: Slot [BAT0] (battery present)
[   13.069844] xhci_hcd 0000:00:14.0: xHCI Host Controller
[   13.069870] xhci_hcd 0000:00:14.0: new USB bus registered, assigned bus number 1
[   13.070971] xhci_hcd 0000:00:14.0: hcc params 0x200077c1 hci version 0x100 quirks 0x0000000000009810
[   13.071313] xhci_hcd 0000:00:14.0: xHCI Host Controller
[   13.071325] xhci_hcd 0000:00:14.0: new USB bus registered, assigned bus number 2
[   13.071343] xhci_hcd 0000:00:14.0: Host supports USB 3.0 SuperSpeed
[   13.071599] usb usb1: New USB device found, idVendor=1d6b, idProduct=0002, bcdDevice= 6.01
[   13.071606] usb usb1: New USB device strings: Mfr=3, Product=2, SerialNumber=1
[   13.071611] usb usb1: Product: xHCI Host Controller
[   13.071614] usb usb1: Manufacturer: Linux 6.1.0-9-amd64 xhci-hcd
[   13.071617] usb usb1: SerialNumber: 0000:00:14.0
[   13.084920] hub 1-0:1.0: USB hub found
[   13.084956] hub 1-0:1.0: 6 ports detected
[   13.089220] usb usb2: New USB device found, idVendor=1d6b, idProduct=0003, bcdDevice= 6.01
[   13.089231] usb usb2: New USB device strings: Mfr=3, Product=2, SerialNumber=1
[   13.089235] usb usb2: Product: xHCI Host Controller
[   13.089239] usb usb2: Manufacturer: Linux 6.1.0-9-amd64 xhci-hcd
[   13.089242] usb usb2: SerialNumber: 0000:00:14.0
[   13.123406] elan_i2c i2c-ELAN0000:00: supply vcc not found, using dummy regulator
[   13.141128] hub 2-0:1.0: USB hub found
[   13.141186] hub 2-0:1.0: 1 port detected
[   13.241129] elan_i2c i2c-ELAN0000:00: Elan Touchpad: Module ID: 0x003d, Firmware: 0x0004, Sample: 0x0004, IAP: 0x0007
[   13.242028] input: Elan Touchpad as /devices/pci0000:00/80860F41:00/i2c-0/i2c-ELAN0000:00/input/input6
[   13.263051] snd_hda_intel 0000:00:1b.0: bound 0000:00:02.0 (ops i915_audio_component_bind_ops [i915])
[   13.343858] usb 1-3: new high-speed USB device number 2 using xhci_hcd
[   13.358624] gsmi: no gsmi handler in firmware
[   13.383884] input: HDA Intel PCH HDMI/DP,pcm=3 as /devices/pci0000:00/0000:00:1b.0/sound/card0/input7
[   13.384269] input: HDA Intel PCH HDMI/DP,pcm=7 as /devices/pci0000:00/0000:00:1b.0/sound/card0/input8
[   13.488759] cryptd: max_cpu_qlen set to 1000
[   13.527084] usb 1-3: New USB device found, idVendor=04f2, idProduct=b462, bcdDevice= 0.06
[   13.527096] usb 1-3: New USB device strings: Mfr=3, Product=1, SerialNumber=2
[   13.527101] usb 1-3: Product: Chromebook HD Camera
[   13.527104] usb 1-3: Manufacturer: Generic_C
[   13.527108] usb 1-3: SerialNumber: 0x0001
[   13.527232] atmel_mxt_ts i2c-ATML0001:00: supply vdda not found, using dummy regulator
[   13.527332] atmel_mxt_ts i2c-ATML0001:00: supply vdd not found, using dummy regulator
[   13.541843] cfg80211: Loading compiled-in X.509 certificates for regulatory database
[   13.542382] cfg80211: Loaded X.509 cert 'benh@debian.org: 577e021cb980e0e820821ba7b54b4961b8b4fadf'
[   13.542877] cfg80211: Loaded X.509 cert 'romain.perier@gmail.com: 3abbc6ec146e09d1b6016ab9d6cf71dd233f0328'
[   13.543369] cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
[   13.557688] platform regulatory.0: firmware: direct-loading firmware regulatory.db
[   13.560401] platform regulatory.0: firmware: direct-loading firmware regulatory.db.p7s
[   13.576381] iTCO_vendor_support: vendor-support=0
[   13.584056] atmel_mxt_ts i2c-ATML0001:00: __mxt_read_reg: i2c transfer failed (-121)
[   13.584155] atmel_mxt_ts i2c-ATML0001:00: mxt_bootloader_read: i2c recv failed (-121)
[   13.584163] atmel_mxt_ts i2c-ATML0001:00: Trying alternate bootloader address
[   13.584283] atmel_mxt_ts i2c-ATML0001:00: mxt_bootloader_read: i2c recv failed (-121)
[   13.618674] Intel(R) Wireless WiFi driver for Linux
[   13.620770] atmel_mxt_ts: probe of i2c-ATML0001:00 failed with error -121
[   13.634284] iwlwifi 0000:01:00.0: firmware: direct-loading firmware iwlwifi-7260-17.ucode
[   13.635474] iwlwifi 0000:01:00.0: loaded firmware version 17.3216344376.0 7260-17.ucode op_mode iwlmvm
[   13.639297] iTCO_wdt iTCO_wdt.1.auto: Found a Bay Trail SoC TCO device (Version=3, TCOBASE=0x0460)
[   13.655848] usb 1-4: new full-speed USB device number 3 using xhci_hcd
[   13.656266] iTCO_wdt iTCO_wdt.1.auto: initialized. heartbeat=30 sec (nowayout=0)
[   13.805006] usb 1-4: New USB device found, idVendor=8087, idProduct=07dc, bcdDevice= 0.01
[   13.805018] usb 1-4: New USB device strings: Mfr=0, Product=0, SerialNumber=0
[   13.966413] wl: loading out-of-tree module taints kernel.
[   13.966427] wl: module license 'MIXED/Proprietary' taints kernel.
[   13.966430] Disabling lock debugging due to kernel taint
[   13.973904] wl: module verification failed: signature and/or required key missing - tainting kernel
[   14.099593] iwlwifi 0000:01:00.0: Detected Intel(R) Dual Band Wireless AC 7260, REV=0x144
[   14.155431] iwlwifi 0000:01:00.0: base HW address: cc:3d:82:99:f7:cd, OTP minor version: 0x0
[   14.243971] intel_rapl_common: Found RAPL domain package
[   14.243980] intel_rapl_common: Found RAPL domain core
[   14.370578] ieee80211 phy0: Selected rate control algorithm 'iwl-mvm-rs'
[   15.638094] mc: Linux media interface: v0.10
[   15.667382] alg: No test for fips(ansi_cprng) (fips_ansi_cprng)
[   15.683051] videodev: Linux video capture interface: v2.00
[   15.729223] usb 1-3: Found UVC 1.00 device Chromebook HD Camera (04f2:b462)
[   15.734655] input: Chromebook HD Camera: Chromeboo as /devices/pci0000:00/0000:00:14.0/usb1/1-3/1-3:1.0/input/input9
[   15.734829] usbcore: registered new interface driver uvcvideo
[   15.964623] Bluetooth: Core ver 2.22
[   15.964686] NET: Registered PF_BLUETOOTH protocol family
[   15.964690] Bluetooth: HCI device and connection manager initialized
[   15.964699] Bluetooth: HCI socket layer initialized
[   15.964704] Bluetooth: L2CAP socket layer initialized
[   15.964714] Bluetooth: SCO socket layer initialized
[   16.145375] usbcore: registered new interface driver btusb
[   16.158823] Bluetooth: hci0: Legacy ROM 2.5 revision 8.0 build 2 week 3 2013
[   16.191040] bluetooth hci0: firmware: direct-loading firmware intel/ibt-hw-37.7.10-fw-1.80.2.3.d.bseq
[   16.191063] Bluetooth: hci0: Intel Bluetooth firmware file: intel/ibt-hw-37.7.10-fw-1.80.2.3.d.bseq
[   16.414733] Bluetooth: hci0: Intel BT fw patch 0x57 completed & activated
[   16.516829] Adding 1022972k swap on /dev/mmcblk1p3.  Priority:-2 extents:1 across:1022972k SSFS
[   16.551539] EXT4-fs (mmcblk1p4): re-mounted. Quota mode: none.
[   17.283759] lp: driver loaded but no devices found
[   17.302312] ppdev: user-space parallel port driver
[   17.365016] loop: module loaded
[   17.965588] networking done
[   18.329632] RPC: Registered named UNIX socket transport module.
[   18.329639] RPC: Registered udp transport module.
[   18.329642] RPC: Registered tcp transport module.
[   18.329643] RPC: Registered tcp NFSv4.1 backchannel transport module.
[   18.417848] FS-Cache: Loaded
[   19.098258] input: ACPI Virtual Keyboard Device as /devices/virtual/input/input10
[   20.989221] Bluetooth: BNEP (Ethernet Emulation) ver 1.3
[   20.989231] Bluetooth: BNEP filters: protocol multicast
[   20.989241] Bluetooth: BNEP socket layer initialized
[   20.995182] Bluetooth: MGMT ver 1.22
[   21.016984] NET: Registered PF_ALG protocol family
[   21.323266] traps: plymouthd[193] trap stack segment ip:7fb27c96f939 sp:7fff788f0e10 error:0 in libply.so.5.0.0 (deleted)[7fb27c96e000+e000]
[   21.518957] ACPI: \: failed to evaluate _DSM bf0212f2-788f-c64d-a5b3-1f738e285ade (0x1001)
[   21.518973] ACPI: \: failed to evaluate _DSM bf0212f2-788f-c64d-a5b3-1f738e285ade (0x1001)
[   21.518978] ACPI: \: failed to evaluate _DSM bf0212f2-788f-c64d-a5b3-1f738e285ade (0x1001)
[   21.518983] ACPI: \: failed to evaluate _DSM bf0212f2-788f-c64d-a5b3-1f738e285ade (0x1001)
[   21.518987] ACPI: \: failed to evaluate _DSM bf0212f2-788f-c64d-a5b3-1f738e285ade (0x1001)
[   21.518992] ACPI: \: failed to evaluate _DSM bf0212f2-788f-c64d-a5b3-1f738e285ade (0x1001)
[   21.518997] ACPI: \: failed to evaluate _DSM bf0212f2-788f-c64d-a5b3-1f738e285ade (0x1001)
[   21.819411] ACPI: \: failed to evaluate _DSM bf0212f2-788f-c64d-a5b3-1f738e285ade (0x1001)
[   21.819431] ACPI: \: failed to evaluate _DSM bf0212f2-788f-c64d-a5b3-1f738e285ade (0x1001)
[   21.819437] ACPI: \: failed to evaluate _DSM bf0212f2-788f-c64d-a5b3-1f738e285ade (0x1001)
[   21.819443] ACPI: \: failed to evaluate _DSM bf0212f2-788f-c64d-a5b3-1f738e285ade (0x1001)
[   21.819449] ACPI: \: failed to evaluate _DSM bf0212f2-788f-c64d-a5b3-1f738e285ade (0x1001)
[   21.819454] ACPI: \: failed to evaluate _DSM bf0212f2-788f-c64d-a5b3-1f738e285ade (0x1001)
[   21.819460] ACPI: \: failed to evaluate _DSM bf0212f2-788f-c64d-a5b3-1f738e285ade (0x1001)
[   22.198175] NET: Registered PF_QIPCRTR protocol family
[   25.579221] wlan0: authenticate with dc:16:b2:4d:9e:8a
[   25.579266] wlan0: 80 MHz not supported, disabling VHT
[   25.583355] wlan0: send auth to dc:16:b2:4d:9e:8a (try 1/3)
[   25.589557] wlan0: authenticated
[   25.594686] wlan0: associate with dc:16:b2:4d:9e:8a (try 1/3)
[   25.608329] wlan0: RX AssocResp from dc:16:b2:4d:9e:8a (capab=0x1631 status=0 aid=9)
[   25.624714] wlan0: associated
[   25.735105] IPv6: ADDRCONF(NETDEV_CHANGE): wlan0: link becomes ready
[   25.829736] wlan0: Limiting TX power to 20 (20 - 0) dBm as advertised by dc:16:b2:4d:9e:8a
[   39.872244] fuse: init (API version 7.37)
[   48.597184] Bluetooth: RFCOMM TTY layer initialized
[   48.597203] Bluetooth: RFCOMM socket layer initialized
[   48.597220] Bluetooth: RFCOMM ver 1.11

**Comment by WeirdTreeThing** [comment]
You'll need the kernel from [https://elly.rocks/tmp/BUILDROOT/](https://elly.rocks/tmp/BUILDROOT/).

**Comment by shur601** [comment]
Tell me, please, what to download from this and how to install?

**Comment by WeirdTreeThing** [comment]
Download the second file in that list and install with `sudo dpkg -i <kernel-file-name.deb>`

**Comment by shur601** [comment]
Thank you. I will try

**Comment by shur601** [comment]
Great! It works. Thanks!
